essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/VideoCodecBase.cs

```diff
@@ -1407,9 +1407,9 @@ private string UpdateCallStatusXSig()
 
 				arrayIndex += offset;
 				stringIndex += maxStrings;
-				digitalIndex++;
+                digitalIndex += maxDigitals;
 			}
-			while (digitalIndex < maxCalls)
+			while (digitalIndex < maxCalls * offset)

```

Comment:
I'm wondering if this should be `while (arrayIndex < maxCalls * offset)`. That would be more in keeping with the way the `UpdateMeetingsXSig` works. It might be six of one, half dozen of the other...but consistency is probably good.